packages/react/src/PricingOptions/PricingOptions.tsx

```diff
@@ -404,15 +407,21 @@ const PricingOptionsFeatureList = forwardRef<HTMLDivElement, PricingOptionsFeatu
 
     const {isMedium: isRegular, isXLarge: isWide} = useWindowSize()
 
+    let FeatureListTitle = () => (
+      <PricingOptions.FeatureListTitle>{pricingOptionsDefaultFeatureListTitle}</PricingOptions.FeatureListTitle>
+    )
```

Comment:
You could simplify this slightly to:

```suggestion
    let FeatureListTitle = <PricingOptions.FeatureListTitle>{pricingOptionsDefaultFeatureListTitle}</PricingOptions.FeatureListTitle>
```